update-agent-loader/Cargo.toml

```diff
@@ -0,0 +1,45 @@
+[package]
+name = "update-agent-loader"
+version = "0.1.0"
+edition = "2021"
+authors = ["Worldcoin <engineering@worldcoin.org>"]
+description = "A loader for the update-agent"
+license = "MIT OR Apache-2.0"
+rust-version = "1.75"
+
+[workspace]
+
+# Enable unstable features needed for compilation
+[package.metadata.rust-analyzer]
+rustc_private = true
+
+[features]
+default = []
+# The allow_http feature enables HTTP URLs for testing purposes
+# This feature cannot be enabled in release mode due to a compile-time check
+allow_http = []
+
+[dependencies]
+reqwest = { version = "0.11.4", features = ["blocking", "rustls-tls-native-roots"], default-features = false }
```

Comment:
Use request from the security utils crate to ensure we are pinning certs properly